pages/signup.tsx

```diff
@@ -1,66 +1,123 @@
-            id='email'
-            type='email'
-            placeholder='E-Mail'
-            value={email}
-            onChange={e => setEmail(e.target.value)}
-          />
+      <main className='flex h-screen w-screen'>
+        <div className='flex h-screen  w-[100%] flex-col items-center justify-center bg-gray-100 lg:w-[50%]'>
+          {/* Now you can decide different login ways - primarily I am giving google and twiiter for more add it yourself. */}
+          <div className='h-full w-full  bg-white p-4 md:shadow-lg lg:h-[650px] lg:w-[450px]'>
+            <h1 className='p-2 text-3xl font-semibold'>Create Account ⚡</h1>
+
+            <form
+              onSubmit={e => {
+                e.preventDefault();
+                handleSignup();
+              }}
+              className='mt-4 flex flex-col space-y-4'
+            >
+              <div className='flex w-[90%] flex-col space-y-2'>
+                <p>Email</p>
+                <input
+                  type='text'
+                  placeholder='example@gmail.com'
+                  required
+                  className='input-field outline-none'
+                  value={email}
+                  onChange={e => setEmail(e.target.value)}
+                />
+              </div>
+              <div className='flex w-[90%] flex-col space-y-2'>
+                <p>Username</p>
+                <input
+                  type='text'
+                  placeholder='example'
+                  required
+                  className='input-field outline-none'
+                  value={username}
+                  onChange={e => setUsername(e.target.value)}
+                />
+              </div>
+              <div className='flex w-[90%] flex-col space-y-2 '>
+                <p>Password</p>
+                <input
+                  type='password'
+                  required
+                  className='input-field outline-none'
+                  placeholder='*****'
+                  value={password}
+                  onChange={e => setPassowrd(e.target.value)}
+                />
+              </div>
+              <div className='flex w-[90%] flex-col space-y-2 '>
+                <p>Confirm Password</p>
+                <input
+                  type='password'
+                  required
+                  className='input-field outline-none'
+                  placeholder='*****'
+                  value={confirmPass}
+                  onChange={e => setConfirmPass(e.target.value)}
+                />
+              </div>
+            </form>
+            <button
+              type='submit'
+              onClick={handleSignup}
+              className='my-4 flex  w-[90%] cursor-pointer items-center  justify-center rounded-md bg-blue-600 py-2 font-semibold text-white hover:bg-blue-500'
+            >
+              Login
+            </button>
+            <p className=' font-semibold'>
+              Already have an account ?
```

Comment:
there's still a space before the question mark. I believe this is the third time I said this

Reply:
Really sorry I assumed you are mentioning padding on the y-axis I will correct it now.